Plugins/Geant4/include/Acts/Plugins/Geant4/Geant4Converters.hpp

```diff
@@ -84,38 +91,38 @@ struct Geant4ShapeConverter {
   /// @param g4Tubs a Geant4 tube shape
   ///
   /// @return an Acts Cylinder bounds object, and thickness
-  std::tuple<std::shared_ptr<CylinderBounds>, ActsScalar> cylinderBounds(
+  std::tuple<std::unique_ptr<CylinderBounds>, ActsScalar> cylinderBounds(
```

Comment:
My thinking here was to create what the surface constructor needs ... what's the motivation to change to `unqiue_ptr` here?

Reply:
Maybe I went a bit overboard with this. My thinking was if we ever need a unique ptr with surface bounds it will not be possible with this interface in a straight forward way. For surface bounds it looks like we never do this anyways but for example `DetectorVolume` will take a volume bounds as a unique prt. So I think it makes sense to stay consistent

Reply:
I see you point, the fact that `Surface` objects have shared pointers as bounds compared to the `Volume` objects that container unique bounds, is the fact that indeed when you have a.g. a straw detector with 100 thousands of straws they all could have the same bounds ...

Reply:
sure I can change that back. I think in principle factories should be using `std::unique_ptr` as a return value because you can always move them to be `std::shared_ptr` but not the other way around. if we only use `std::shared_ptr` of these it is fair to build them in that way in the first place